libmount/src/tab_update.c

```diff
@@ -1004,6 +1004,25 @@ int mnt_update_already_done(struct libmnt_update *upd, struct libmnt_lock *lc)
 	return rc;
 }
 
+int mnt_update_emit_event(struct libmnt_update *upd)
+{
+	char *filename;
+	int fd;
+
+	if (!upd || !upd->filename)
+		return -EINVAL;
+
+	if (asprintf(&filename, "%s.event", upd->filename) <= 0)
```

Comment:
The different manpages of `asprintf()` do not say it sets `errno`.

Reply:
OK, I will fit it.